MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,276 @@
+    public DateTime JoinDate { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Validation mode
+
+The [ValidationMode] determines when the value should be validated.
+
+The supported validation modes are as follows:
+
+* LostFocus
+* PropertyChanged
+* Manual
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm" 
+        ValidationMode="LostFocus">
+    </dataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.ValidationMode = DataFormValidationMode.LostFocus;
+
+{% endhighlight %}
+{% endtabs %}
+
+#### LostFocus
+
+If the validation mode is `LostFocus`, the value will be validated when the editor lost its focus. By default the [ValidationMode] is `LostFocus`
+
+#### PropertyChanged
+
+If the validation mode is `PropertyChanged`, the value will be validated immediately when it is changed.
+
+#### Manual
+
+If the validation mode is `Manual`, the value should be validated manually by calling the [SfDataForm.Validate] or [SfDataForm.Validate(new List<string>())] method.
+
+The following code validates the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.Validate();
+
+{% endhighlight %}
+{% endtabs %}
+
+To validate the specific list of property value, pass the property names as argument.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.Validate(new List<string>() {"FirstName", "Age" });
+
+{% endhighlight %}
+{% endtabs %}
+
+You can determine whether the data form or property is valid or not by using the `Validate` method.
+
+{% tabs %}
+{% highlight C# %}
+
+        bool isValid = this.dataForm.Validate();
+
+        List<string> propertyNames = new List<string>();
+        propertyNames.Add("FirstName");
+        bool isPropertyValid = this.dataForm.Validate(propertyNames);
+
+{% endhighlight %}
+{% endtabs %}
+
+If the data form or property is valid, `true` will be returned. Or else `false` will be returned.
+
+## Valid message
+
+If the values are correct, you can show the [ValidMessage] as like error message, the valid message will also be displayed at the bottom of the editor.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataFormDisplayOptions(ValidMessage = "Password strength is good")]
+    [Required(ErrorMessage = "Please enter the password")]
+    [RegularExpression(@"^(?=.*[a-z])(?=.*[A-Z])[a-zA-Z\d]{8,}$", ErrorMessage = "A minimum 8-character password should contain a combination of uppercase and lowercase letters.")]
+    public string Password { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Validate the data form
+
+You can get the validation details of the all the editors of data form using the [ValidateForm] event of the data form.
+
+Note: This event will be raised once after the manual validation call using [SfDataForm.Validate()] method.
```

Comment:
**Note:** This event will be raised once after the manual validation call using the [SfDataForm.Validate()] method.

Reply:
content changed